src/lwc/geFormSection/__tests__/geFormSection.test.js

```diff
@@ -0,0 +1,65 @@
+import { createElement } from 'lwc';
+import GeFormSection from 'c/geFormSection';
+
+import section from './data/section.json';
+import sectionWithElevateWidget from './data/sectionWithElevateWidget.json';
+
+jest.mock('c/geFormField');
+jest.mock('c/geFormWidgetTokenizeCard');
+
+describe('c-ge-form-section', () => {
+    afterEach(() => {
+        clearDOM();
+        jest.clearAllMocks();
+    });
+
+    describe('rendered elements', () => {
+        it('should render expandable section with provided label', async () => {
+            const element = create(section);
+
+            await flushPromises();
+            const expandableSectionElements = shadowSelectorAll(element, 'c-util-expandable-section');
+            expect(expandableSectionElements.length).toBe(1);
+
+            const sectionLabel = shadowSelector(expandableSectionElements[0], 'span');
+            expect(sectionLabel.innerHTML).toBe('Donor Information');
+        });
+
+        it('should render 3 form fields', async () => {
+            const element = create(section);
+
+            await flushPromises();
+            const formFieldElements = shadowSelectorAll(element, 'c-ge-form-field');
+            expect(formFieldElements.length).toBe(3);
+        });
+
+        it('should render 3 form fields and the elevate widget', async () => {
```

Comment:
One thing I'd like to see in this block of tests is one where we listen for the "registerPaymentWidgetEvent". A negative case where we assert that the event isn't fired, and a positive case where it does.

Reply:
Great suggestion! Can do!